src/main/java/org/jitsi/srtp/crypto/OpenSslAesGcmAuthOnlyCipherSpi.java

```diff
@@ -0,0 +1,389 @@
+/*
+ * Copyright @ 2016 - present 8x8, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.srtp.crypto;
+
+import java.security.*;
+import java.security.spec.*;
+import java.util.*;
+import javax.crypto.*;
+import javax.crypto.spec.*;
+
+/**
+ * AES-GCM auth-only Cipher SPI.
+ * Can only be used for "decryption".
+ */
+public class OpenSslAesGcmAuthOnlyCipherSpi
```

Comment:
Is the plan to merge this back into OpenSslAesCipherSpi or to keep it as a separate class?

Reply:
It's using a sufficiently different API, with a sufficiently restricted use case, that I'm inclined to keep it as a separate class.